src/Features/Core/Portable/EmbeddedLanguages/StackFrame/StackFrameLexer.cs

```diff
@@ -479,5 +509,22 @@ private static StackFrameKind GetKind(VirtualChar ch)
 
         private static bool IsNumber(VirtualChar ch)
             => ch.Value is >= '0' and <= '9';
+
+        private readonly record struct Language(string At, string In, string Line);
+        private static readonly ImmutableArray<Language> s_languages = ImmutableArray.Create(
+            new Language("in ", " in ", "line "), // en
+            new Language("v ", " v ", "řádek "), // cs
```

Comment:
>cs [](http://example.com/codeflow?start=52&length=2)

I like how Czech is the second after EN :)

Reply:
😂❤️